docs/guides/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_in_dbfs.rst

```diff
@@ -0,0 +1,136 @@
+
+    .. code-block:: yaml
+
+        expectations_store_name: expectations_store
+
+        stores:
+            expectations_store:
+                class_name: ExpectationsStore
+                store_backend:
+                    class_name: TupleFilesystemStoreBackend
+                    base_directory: expectations/
+
+
+3. **Update your configuration file to include a new store for Expectations results on your filesystem**
+
+    In the example below, Expectations Store is being set to ``shared_expectations_filesystem_store`` with the ``base_directory`` set to ``shared_expectations/``.
+
+    .. code-block:: yaml
+
+        expectations_store_name: shared_expectations_filesystem_store
+
+        stores:
+            shared_expectations_filesystem_store:
+                class_name: ExpectationsStore
+                store_backend:
+                    class_name: TupleFilesystemStoreBackend
+                    base_directory: shared_expectations/
+
+
+
+4. **Confirm that the location has been updated by running** ``great_expectations store list``.
+
+    Notice the output contains two Expectation stores: the original ``expectations_store`` on the local filesystem and the ``shared_expectations_filesystem_store`` we just configured.  This is ok, since Great Expectations will look for Expectations in the ``shared_expectations/`` folder as long as we set the ``expectations_store_name`` variable to ``shared_expectations_filesystem_store``.  The config for ``expectations_store`` can be removed if you would like.
+
+    .. code-block:: bash
+
+        great_expectations store list
+
+        2 Stores found:
+
+        - name: expectations_store
+        class_name: ExpectationsStore
+        store_backend:
+            class_name: TupleFilesystemStoreBackend
+            base_directory: expectations/
+
+        - name: shared_expectations_filesystem_store
+        class_name: ExpectationsStore
+        store_backend:
+            class_name: TupleFilesystemStoreBackend
+            base_directory: shared_expectations/
+
+
+5. **Confirm that Expectations can be read from the new storage location by running** ``great_expectations suite list``.
+
+    .. code-block:: bash
+
+        great_expectations suite list
+
+        1 Expectation Suite found:
+            - npi_expectations
+
+
+Additional Notes
+----------------
+
+- For best practices, we highly recommend that you store Expectations in a version-control system like Git. The JSON format of Expectations will allow for informative diff-statements and effective tracking of modifications. In the example below, 2 changes have been made to ``npi_expectations``.  The Expectation ```expect_table_column_count_to_equal`` was been changed from ``330`` to ``333`` to ``331``.
```

Comment:
Typo: was been changed > has been changed / was changed

Reply:
Hmmm do we need this comment at all? Seems like it's explaining something that should be explained in a more general place.